tests/integration.rs

```diff
@@ -617,10 +617,10 @@ Caused by:
             "http-ts",
             Some("https://github.com/fermyon/spin-js-sdk"),
             None,
-            &["js2wasm"],
+            &[],
             prebuild,
             HashMap::default(),
-            "Hello from TS-SDK",
+            "hello universe",
```

Comment:
Just curious. Why was this changed?

Reply:
Bit of a shame - I think the previous versions were better, but this isn't the place to discuss that I suppose.